prettier.config.js

```diff
@@ -1,6 +1,8 @@
 /** @type {import('prettier').Config & import('prettier-plugin-tailwindcss').PluginOptions} */
 const config = {
   plugins: ["prettier-plugin-tailwindcss"],
+  endOfLine: "lf",

```

Comment:
I'm pretty sure prettier respects the [`.editorconfig`](https://editorconfig.org/) file. I think settings that are supported should be configured in `.editorconfig`